Scripts/Python/ki/xKIChat.py

```diff
@@ -627,8 +641,9 @@ def AddChatLine(self, player, message, cFlags, forceKI=True):
                         chatArea.insertStringW(chatMessageFormatted[lastInsert:start], censorLevel=censorLevel)
 
                     lastInsert = end
+                    
                     chatArea.insertColor(mentionColor)
-                    chatArea.insertStringW(mention, censorLevel=censorLevel, urlDetection=False)
+                    chatArea.insertStringW(f"{'*' if self.chatTextColor else ''}{mention}{'*' if self.chatTextColor else ''}", censorLevel=censorLevel, urlDetection=False)
```

Comment:
What if the player's name starts and ends with asterisks?

Reply:
Now with 100% more asterisks! 🤷 

More seriously, the other options I could think of were to either ignore mentions entirely or to have a prefix of like "[PING]" or whatever... neither of which I liked very much. Putting some kind of noticeable markers (like asterisks) around the name seemed like the most reasonable thing I could do that was similar to the existing functionality that used color difference.

Reply:
I kind of like the [PING] prefix lol

Reply:
I was thinking about adding mention support for messages coming from Buddies and Neighbors. That seemed like an oversight that it doesn't currently search for mentions in those sources. In that case, though, you'd end up doubling up on prefixes "[PING][BUDDIES]", which seems... rather unfortunate.

Reply:
I'd love some more perspectives and opinions on the mention/ping topic and the PR as a whole, if possible.